persistence/src/main/scala/com/lightbend/lagom/javadsl/persistence/AggregateEventTag.scala

```diff
@@ -24,29 +24,30 @@ object AggregateEventTag {
     new AggregateEventTag(eventType, tag)
 
   /**
-   * Create a sharded aggregate event tag.
+   * Create an aggregate event shards tagger.
    *
    * This is a convenience function that uses the name of the class as the tag name. Note that if the class name
    * changes, the tag name must be retained, and so this method will no longer be suitable for use.
    *
-   * A tag will be selected based on a hash function that combines the `entityId` and `numShards`.
+   * Events that return this will be tagged with a tag that is based on a hash of the events persistence ID, modulo
+   * the number of shards.
    *
    * The <code>numShards</code> should be selected up front, and shouldn't change. If it does change, events for the
    * same entity will be produced by different event streams and handled by different shards in the read side
    * processor, leading to out of order event handling.
    *
    * @param eventType The type of the event.
    * @param numShards The number of shards.
-   * @param entityId The ID of the entity.
-   * @return The aggregate event tag.
+   * @return The aggregate event shards tagger.
    */
-  def shard[Event <: AggregateEvent[Event]](eventType: Class[Event], numShards: Int, entityId: String): AggregateEventTag[Event] =
-    shard(eventType, eventType.getName, numShards, entityId)
+  def sharded[Event <: AggregateEvent[Event]](eventType: Class[Event], numShards: Int): AggregateEventShards[Event] =
```

Comment:
If we do that, we should also deprecate the equivalent `of` method and remove the notes in the docs about using the class name.